### ISSUE TITLE
Reporter JSON being silently ignored. 

### ISSUE THREAD

**Issue by Emmanux** [DevExpress/testcafe#1881]
### Are you requesting a feature or reporting a bug?
Reporting a bug.

### What is the current behavior?
testcafe ignores the --reporter=json option. 

### What is the expected behavior?
To use that built-in module.

### How would you reproduce the current behavior (if this is a bug)?
```sudo /usr/bin/docker run --rm   -e CI=1  -v `pwd`/tests:/tests   --net=dockernet   testcafe/testcafe   'chromium --no-sandbox --kiosk --disable-translate --disable-infobars' --reporter json   /tests/test.js```

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```js

```

### Specify your

* operating system:
* testcafe version:
* node.js version:

**Comment by AndreyBelym** [comment]
Hello @Emmanux !

There was a breaking change in `0.18.0`. The `--reporter` key was renamed to `--reporters` due to support of multiple reporters. We have to fix docs and allow to use the old version (maybe with some deprecation warning).

You can use short version of this key (`-r`) as a workaround:
```
sudo /usr/bin/docker run --rm -e CI=1 -v `pwd`/tests:/tests --net=dockernet testcafe/testcafe 'chromium --no-sandbox --kiosk --disable-translate --disable-infobars' -r json /tests/test.js
```

**Comment by Emmanux** [comment]
Thanks Andrey, I'm looking forward to the new documentation.

**Comment by AlexanderMoskovkin** [comment]
Hi @Emmanux,

We've reviewed our sources, it seems this behavior was changed by a mistake. We should revert it back soon. Please use the `-r` option as workaround.

**Comment by churkin** [comment]
fixed in `0.18.1`
